Error: Cannot find module 'mui-toolpad/dist/index.js'

### Duplicates

- [X] I have searched the existing issues

### Latest version

- [X] I have tested the latest version

### Steps to reproduce 🕹

Steps:

1. yarn install
2. yarn dev
3. yarn toolpad dev test/integration/backend-basic/fixture --dev 


### Current behavior 😯

It's throwing an error.

```bash
$ yarn toolpad dev test/integration/backend-basic/fixture --dev 
node:internal/modules/cjs/loader:1080
  throw err;
  ^

Error: Cannot find module '/Users/dan/test/mui-toolpad/dist/index.js'
    at Module._resolveFilename (node:internal/modules/cjs/loader:1077:15)
    at Module._load (node:internal/modules/cjs/loader:922:27)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:81:12)
    at node:internal/main/run_main_module:23:47 {
  code: 'MODULE_NOT_FOUND',
  requireStack: []
}

Node.js v18.16.1
```

### Expected behavior 🤔

For there to be no error

### Context 🔦

Was following the contributing.md guide to run `examples/qr-generator` and got an error. Then I Realized I got the same error for `est/integration/backend-basic/fixture` too and that's easier to report in the issue.

### Your environment 🌎

<details>
  <summary><code>npx @mui/envinfo</code></summary>

```
  System:
    OS: macOS 13.4
  Binaries:
    Node: 18.16.1 - ~/.nvm/versions/node/v18.16.1/bin/node
    Yarn: 3.6.1 - ~/.nvm/versions/node/v18.16.1/bin/yarn
    npm: 9.5.1 - ~/.nvm/versions/node/v18.16.1/bin/npm
  Browsers:
    Chrome: 116.0.5845.187 (using chrome)
    Edge: Not Found
    Safari: 16.5


```
</details>

--- a/package.json
+++ b/package.json
@@ -36,7 +36,7 @@
     "test:argos": "node ./scripts/pushArgos.mjs",
     "test": "lerna run test",
     "check-types": "lerna run check-types",
-    "toolpad": "toolpad",
+    "toolpad": "node packages/toolpad-app/cli.js",
     "jsonSchemas": "tsx ./scripts/docs/generateJsonSchemas.ts",
     "test:rest:start": "tsx ./scripts/restTestServer.ts"
   },

